docs/dictionary/control_st/return.lcdoc

```diff
@@ -2,39 +2,142 @@ Name: return
 
 Type: control structure
 
-Syntax: return <value> 
+Syntax: return [ value | error ] <value>
```

Comment:
BTW I can't help wondering if the socket ID should be a single parameter in syntax declarations with detail of `IP:port|ID` in the param description to avoid the special case `| ID` having to be dealt with by anything parsing syntax declarations. I'm happy to make this change to the relevant docs and remove the condition from the syntax checker if people agree. There's only 4 docs that separate `socketID` into the components as separate parameters and it could give users the mistaken impression that you can declare a socket ID as `"127.0.0.1":"80"|"myserver"`.